airbyte-cdk/java/airbyte-cdk/typing-deduping/src/test/kotlin/io/airbyte/integrations/base/destination/typing_deduping/DefaultTyperDeduperTest.kt

```diff
@@ -913,11 +913,13 @@ class DefaultTyperDeduperTest {
                     "overwrite_ns",
                     "overwrite_stream"
                 ),
-                mock(),
                 DestinationSyncMode.OVERWRITE,
                 mock(),
                 mock(),
-                mock()
+                mock(),
+                0,
```

Comment:
for a later stacked PR: I think we need to add generation-related logic into DefaultTyperDeduper? so we'll want more test cases in this class...